app/src/pages/model/ModelPage.tsx

```diff
@@ -69,14 +69,22 @@ export function ModelPage(_props: ModelPageProps) {
         >
           <Tabs>
             <TabPane name="All" key="all">
-              <Accordion>
-                <AccordionItem title="Embeddings" id="embeddings">
-                  <ModelEmbeddingsTable model={data} />
-                </AccordionItem>
-                <AccordionItem title="Dimensions" id="dimensions">
-                  <ModelSchemaTable model={data} />
-                </AccordionItem>
-              </Accordion>
+              <DisclosureGroup
+                defaultExpandedKeys={["embeddings", "dimensions"]}
+              >
+                <Disclosure id="embeddings">
+                  <DisclosureTrigger>Embeddings</DisclosureTrigger>
+                  <DisclosurePanel>
+                    <ModelEmbeddingsTable model={data} />
+                  </DisclosurePanel>
+                </Disclosure>
+                <Disclosure id="dimensions">
```

Comment:
There's a missing border here at the bottom of the table.
<img width="2520" alt="Screenshot 2025-01-02 at 7 40 09 PM" src="https://github.com/user-attachments/assets/3f657876-8d37-4982-ba78-26cfabae5f96" />

You can run this via `pnpm run dev:embeddings`

Reply:
I ran through all of the embedding usages before pushing. I honestly can't tell which border is missing from this screenshot 😅

Reply:
I'd expect there to be a visual separation between the table pagination and the row that says "Dimensions"

Reply:
<img width="1231" alt="image" src="https://github.com/user-attachments/assets/87b39b02-7dcb-47a7-a30c-c69cd1f54902" />

Disclosure group will now add a border between disclosures, as long as it is expanded and not the last item